### PR TITLE
Update docs content

### PR DESCRIPTION
**Pull request by julianrubino**
Related: https://github.com/zeta-chain/docs/issues/358

Refactor Validate docs.

- Renamed Validate -> Operate.
- Restructured the whole Validate section.
- Add readTime and readType to docs.
- Replaced ' with ’ for consistency.

### CODE REVIEW

**Review by julianrubino — COMMENTED** [review]
ci

**Comment by julianrubino** [comment]
ci

File: src/pages/nodes/_meta.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+  "index": {
+    "title": "Operate",
+    "description": "Become one of the important pieces of ZetaChain operation by validating transactions for the chain. It's easy to get setup and started on our testnet and mainnet."
```
Comment (CharlieMc0) [inline src/pages/nodes/_meta.json:4]: Become an important part of the ZetaChain network by operating an node.

File: src/pages/nodes/index.mdx [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
-title: Validate
-description: Become one of the important pieces of ZetaChain operation by validating transactions for the chain. It’s easy to get setup and started on our testnet and mainnet.
-heroImgUrl: /img/pages/validate.svg
+title: Operate
+description: Become one of the important pieces of ZetaChain operation by running your own nodes. It’s easy to get setup and started on our Testnet and Mainnet.
```
Comment (CharlieMc0) [inline src/pages/nodes/index.mdx:3]: "Become one of the important pieces of ZetaChain operation by validating transactions for the chain." - > "Become an important part of the ZetaChain network by operating an node. It's easy to get started on Testnet or Mainnet."

File: src/pages/developers/services/pyth.mdx [hunk 3]
```diff
@@ -18,7 +18,7 @@ Pyth contract addresses:
 
 | Network      | Address                                                                                                                                    |
 | :----------- | :----------------------------------------------------------------------------------------------------------------------------------------- |
-| Mainnet Beta | [0x2880aB155794e7179c9eE2e38200202908C17B43](https://zetachain.blockscout.com/address/0x2880aB155794e7179c9eE2e38200202908C17B43)          |
+| Mainnet      | [0x2880aB155794e7179c9eE2e38200202908C17B43](https://zetachain.blockscout.com/address/0x2880aB155794e7179c9eE2e38200202908C17B43)          |
```
Comment (fadeev) [inline src/pages/developers/services/pyth.mdx:21]: As far as I know the "Beta" label is still relevant.

File: src/pages/nodes/requirements.mdx [hunk 4]
```diff
@@ -8,40 +8,37 @@ These are the recommended specs for a ZetaChain node.
 
 | Node Type       | CPU    | Memory       | Data Disk         |
 | :-------------- | :----- | :----------- | :---------------- |
-| Observer Signer | 8 CPUs | 32 GB Memory | 300+ GB Data Disk |
 | Core Validator  | 4 CPUs | 16 GB Memory | 300+ GB Data Disk |
+| Fullnode        | 4 CPUs | 12 GB Memory | 300+ GB Data Disk |
```
Comment (CharlieMc0) [inline src/pages/nodes/requirements.mdx:12]: `fullnode` or `full node` ?

Comment (fadeev) [inline src/pages/nodes/requirements.mdx:12]:
```suggestion
| Full node        | 4 CPUs | 12 GB Memory | 300+ GB Data Disk |
```

**Review by CharlieMc0 — COMMENTED** [review]
They need to be put into a subfolder with a title. 
There should be a "Start Here" Section with Requirements, Setup, and sync a node
If you look at the other sections there are examples of how to setup the meta file.

**Comment by julianrubino** [comment]
Addressed all comments, thanks for reviewing

**Review by CharlieMc0 — APPROVED** [review]

**Pull request merged**